packages/wonder-blocks-core/util/util.js

```diff
@@ -59,8 +60,22 @@ export function processStyleList<T: Object>(
         }
     });
 
+    const inlineStylesObject = Object.assign({}, ...inlineStyles);
+
+    // TODO(somewhatabstract): When aphrodite no longer puts "!important" on
+    // all the styles, remove this <ADD JIRA ISSUE HERE IF THIS PASSES REVIEW>
+    // If we're not snapshotting styles, let's create a class for the inline
+    // styles so that they can apply to the element even with aphrodite's
+    // use of !important.
+    if (inlineStyles.length > 0 && !shouldInlineStyles) {
+        const inlineStylesStyleSheet = StyleSheet.create({
+            inlineStyles: inlineStylesObject,
+        });
+        stylesheetStyles.push(inlineStylesStyleSheet.inlineStyles);
+    }
+
     return {
-        style: Object.assign({}, ...inlineStyles),
+        style: shouldInlineStyles ? inlineStylesObject : {},
```

Comment:
If we're not inlining styles, then give an empty object here as all our inline styles are now in a class.